app/renderer/components/messageBox.js

```diff
@@ -0,0 +1,165 @@
+const Dialog = require('../../../js/components/dialog')
+const Button = require('../../../js/components/button')
+const SwitchControl = require('../../../js/components/switchControl')
+const appActions = require('../../../js/actions/appActions')
+const KeyCodes = require('../../common/constants/keyCodes')
+const config = require('../../../js/constants/config')
+const {makeImmutable} = require('../../common/state/immutableUtil')
+const {StyleSheet, css} = require('aphrodite')
+const commonStyles = require('./styles/commonStyles')
+
+class MessageBox extends ImmutableComponent {
+  constructor () {
+    super()
+    this.onClick = this.onClick.bind(this)
+    this.onKeyDown = this.onKeyDown.bind(this)
+    this.onSuppressChanged = this.onSuppressChanged.bind(this)
+  }
+
+  componentWillMount () {
+    document.addEventListener('keydown', this.onKeyDown)
+  }
+  componentWillUnmount () {
+    document.removeEventListener('keydown', this.onKeyDown)
+  }
+
+  get tabId () {
+    return this.props.tabId || ''
+  }
+
+  get title () {
+    const msgBoxTitle = (this.props.detail && this.props.detail.get('title')) || ''
+    return msgBoxTitle.replace(config.braveExtensionId, 'Brave')
+  }
+
+  get message () {
+    return (this.props.detail && this.props.detail.get('message')) || ''
+  }
+
+  get buttons () {
+    return (this.props.detail && this.props.detail.get('buttons')) || makeImmutable(['OK'])
+  }
+
+  get cancelId () {
+    return this.props.detail && this.props.detail.get('cancelId')
+  }
+
+  get suppress () {
+    return (this.props.detail && this.props.detail.get('suppress')) || false
+  }
+
+  get showSuppress () {
+    return (this.props.detail && this.props.detail.get('showSuppress')) || false
+  }
+
+  onClick (e) {
+    e.stopPropagation()
+  }
+
+  onKeyDown (e) {
+    if (this.props.isActive) {
+      switch (e.keyCode) {
+        case KeyCodes.ENTER:
+          this.onDismiss(this.tabId)
+          break
+        case KeyCodes.ESC:
+          this.onDismiss(this.tabId, this.cancelId)
+          break
+      }
+    }
+  }
+
+  onSuppressChanged (e) {
+    const detail = this.props.detail.toJS()
+    detail.suppress = !detail.suppress
+    appActions.tabMessageBoxUpdated(this.tabId, detail)
+  }
+
+  onDismiss (tabId, buttonId) {
+    const response = {
+      suppress: this.suppress,
+      result: true
+    }
+
+    if (typeof this.cancelId === 'number') {
+      response.result = buttonId !== this.cancelId
+    }
+
+    appActions.tabMessageBoxDismissed(this.tabId, response)
+  }
+
+  get messageBoxButtons () {
+    const buttons = []
+
+    for (let index = (this.buttons.size - 1); index > -1; index--) {
+      buttons.push(<Button l10nId={this.buttons.get(index)}
+        className={index === 0 ? 'primaryButton' : 'whiteButton'}
+        onClick={this.onDismiss.bind(this, this.tabId, index)} />)
+    }
+
+    return buttons
+  }
+
+  render () {
+    return <Dialog className='noOutline'>
```

Comment:
Also good catch- I wasn't sure about it at first, because Dialog is not an Aphrodite component... but adding a className using Aphrodite works just fine 👍